libbeat/outputs/kafka/kafka.go

```diff
@@ -51,6 +51,23 @@ var (
 		"gzip":   sarama.CompressionGZIP,
 		"snappy": sarama.CompressionSnappy,
 	}
+
+	kafkaVersions = map[string]sarama.KafkaVersion{
```

Comment:
I was first thinking something is wrong below as some "entries" were missing because the sorting on the right is not based on the version. I would probably sort based on the values on the right side. But that is more my personal preference. Just wanted to note it as I was first confused.